doc/readme/EXAMPLES.md

```diff
@@ -94,6 +94,14 @@ You can render it with
 which is also part of `make doc`.
 The target might not be available if the requirements in `doc/cookbook/requirements.txt` are not satisfied (in particular Sphinx), or if the meta examples are disabled.
 
+Even if all the requirements are satisfied and the following error is visible
```

Comment:
plz try to rephrase this... as clearly if SPHINX_EXECUTABLE is not set then not all the requirements are satisfied...

Reply:
Yes, I have now mentioned that,
1) check CMakeCache.txt to see value of SPHINX_EXECUTABLE.
2) If not set then use cmake.
3) If set, requirements haven't been fulfilled.